images/builder/main.go

```diff
@@ -144,7 +145,7 @@ func runSingleJob(o options, jobName, uploaded, version string, subs map[string]
 
 	cmd := exec.Command("gcloud", args...)
 
-	if o.logDir != "" {
+	if o.logDir != "" && strings.ToLower(os.Getenv("LOG_TO_STDOUT")) != "y" {
```

Comment:
I would've done this in the bash script to avoid setting `--log-dir` at all. I don't think it makes sense for this tool to sometimes not do the thing you told it to.

Reply:
Good point. Fixed!